src/main/java/frc/robot/subsystems/instances/TimeOfFlight.java

```diff
@@ -0,0 +1,23 @@
+package main.java.frc.robot.subsystems.instances;
```

Comment:
I dont think we want this subsystems. the time of flight sensor is used by the magazine subsystem, but probably doesnt warrant its own subsystem. Is there are reasons why we need to manage this as a standalone system?

Reply:
Mr. Pack thought that there would need to be its own subsystem because of how the robot processes it, but I looked at the libraries and it doesn't